src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -2783,7 +2783,16 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
   <data name="Make_class_abstract" xml:space="preserve">
     <value>Make class 'abstract'</value>
   </data>
+  <data name="Inline_0" xml:space="preserve">
+    <value>Inline '{0}'</value>
+  </data>
   <data name="Remove_async_modifier" xml:space="preserve">
     <value>Remove 'async' modifier</value>
   </data>
+  <data name="Keep_0" xml:space="preserve">
+    <value>Keep '{0}'</value>
```

Comment:
these strings seem oddly terse.  i would `Inline '{0}'` and `Inline and remove '{0}'`.

Reply:
or:

`Inline '{0}'` and `Inline and keep '{0}'`, depending on if we want the default to be removal or not.  Personally, i want that to be the default.  I think if i'm inlining, most of the time i wwant to remove the existing method.  And only occasionally do i want to keep it around.